addons/binding/org.openhab.binding.onebusaway/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="onebusaway"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <channel-group-type id="arrivalRange">
+        <label>Range</label>
+        <description>Range for a route event</description>
+        <channels>
+            <channel id="arrival" typeId="arrival" />
+            <channel id="departure" typeId="departure" />
+            <channel id="event" typeId="routeEvent" />
+            <channel id="update" typeId="update" />
+        </channels>
+    </channel-group-type>
+
+    <channel-type id="arrival">
+        <item-type>DateTime</item-type>
+        <label>Arrival Time</label>
+        <description>The arrival time of the event</description>
```

Comment:
I went with event based in part with how the astro binding named things, and because you can add an offset on the channel, so it's not quite the arrival time, but rather time of the event as you have configured it.

The word "bus" is a misnomer; this can and is used for things like light rail in a subway, for instance.

I'm open to a better name, but I haven't come up with one.